frontend/src/tests/import-export.test.js

```diff
@@ -107,6 +115,25 @@ describe("Import/export library functionality", () => {
         // prepare instructor json
         const instructorJson = instructorData.map(prepareMinimal.instructor);
         expect(instructorJson).toMatchSnapshot();
+
+        // ROUND TRIP TEST for prepareData function
+        // create instructor CSV File object
+        const instructorCSV = prepareInstructorData(instructorData, "csv");
+        const workbook = XLSX.read(instructorCSV.fileBits[0], {
+            type: "array",
+        });
+        const sheet = workbook.Sheets[workbook.SheetNames[0]];
+        let dataCSV = XLSX.utils.sheet_to_json(sheet, { header: 1 });
+        const keys = dataCSV.shift();
+        // transform to array of objects
+        dataCSV = dataCSV.map(function (row) {
+            return keys.reduce(function (obj, key, i) {
```

Comment:
`reduce` is rarely the right option in Javascript. This would be much clearer as a for loop.

There is also `Object.fromEntries` that you can look at.